src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/base-table.js

```diff
@@ -1,6 +1,6 @@
 import AlloyEditor from 'alloyeditor';
 
-export default class EzConfigTableBase {
+export default class EzConfgiFixedBase {
```

Comment:
Something is wrong here.

Reply:
no, in 2.5 this class has a bug in the name, please see in src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/base-fixed.js

Reply:
I don't mean the typo.
You are changing the class name not extend the class with `EzConfgiFixedBase`, or it's too little coffee for me?

Reply:
Yes, you are right!, I tested this, and worked even when I changed the name class. I need more coffee :), corrected